lib/errors.js

```diff
@@ -52,6 +52,10 @@ const codes = {
     '%s',
     400
   ),
+  FST_ERR_LISTEN_OPTIONS_INVALID: createError(
+    'FST_ERR_LISTEN_OPTIONS_INVALID',
+    "Invalid listen options: '%s'"
+  ),
```

Comment:
```suggestion
    "Invalid listen options: '%s'",
    500,
    TypeError
  ),
```

Reply:
@Uzlopak Why should we have status code? this error not related to request